string name
string[] parents
string[] children

float32 mean
float32 variance

float32[] mean_scalar # scalar for parents
